runironic.sh

```diff
@@ -42,12 +42,26 @@ crudini --set /etc/ironic/ironic.conf pxe tftp_master_path /shared/tftpboot
 crudini --set /etc/ironic/ironic.conf pxe instance_master_path /shared/html/master_images
 crudini --set /etc/ironic/ironic.conf pxe images_path /shared/html/tmp
 crudini --set /etc/ironic/ironic.conf pxe pxe_config_template \$pybasedir/drivers/modules/ipxe_config.template
+crudini --set /etc/ironic/ironic.conf agent deploy_logs_collect always
+crudini --set /etc/ironic/ironic.conf agent deploy_logs_local_path /shared/log/ironic/deploy
 
 ironic-dbsync --config-file /etc/ironic/ironic.conf upgrade
 
 mkdir -p /shared/html
-/usr/bin/python2 /usr/bin/ironic-conductor > /var/log/ironic-conductor.out 2>&1 &
-/usr/bin/python2 /usr/bin/ironic-api > /var/log/ironic-api.out 2>&1 &
+
+# Remove log files from last deployment
+if [ -d "/shared/log/ironic" ]; then
+   rm -rf /shared/log/ironic/*
+else
+   mkdir -p /shared/log/ironic
+fi
+
+# Make logs available in shared mount
+ln -s /shared/log/ironic/ironic-conductor.log /var/log/ironic/ironic-conductor.log
+ln -s /shared/log/ironic/ironic-api.log /var/log/ironic/ironic-api.log
+
+/usr/bin/python2 /usr/bin/ironic-conductor 2>&1 | tee /shared/log/ironic/ironic-conductor.out &
```

Comment:
I think you should use something like `--log-file /shared/log/ironic/ironic-conductor.log` (let's not use `out` extension, it's for `a.out` files).

Reply:
Using --log-file is a better way to go, changed to use that.